var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -22,21 +22,20 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-import glob
-import os
 from spack import *
 
 
-class SuperluDist(Package):
+class SuperluDist(CMakePackage):
     """A general purpose library for the direct solution of large, sparse,
     nonsymmetric systems of linear equations on high performance machines."""
 
     homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
-    url      = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.1.tar.gz"
+    url      = "https://github.com/xiaoyeli/superlu_dist/archive/v6.0.0.tar.gz"
     git      = "https://github.com/xiaoyeli/superlu_dist.git"
 
     version('develop', branch='master')
     version('xsdk-0.2.0', tag='xsdk-0.2.0')
+    version('6.0.0', '2e3ce927fa5786470dacbdf8c41afb08')
```

Comment:
oh, please use `sha256`, something like 
```
sha256='8311ab1a57f142f8057508d4c360e086fcea32f020b4181fed88fee72554351c'
```

you can do `spack fetch superlu-dist@x.y.z` to check/update others.

Reply:
If github URLs are preferable - we could just use git tags?

```
+    version('6.0.0', 'tag=v6.0.0')
+    version('5.4.0', 'tag=v5.4.0')
```

Reply:
i don't think tags are checksummed in Spack, so IMO it's better to use tarbals.

Reply:
well - my thought here is -  git keeps track of checksums - so spack doesn't need to do that  extra work.. [when using git tags]

Reply:
i was thinking about the case when tags are moved forcefully after creation. Of course, that's not the case for this package.

Reply:
You could add the commit hash rather than the tag. The commit is "secure", the tag isn't

Reply:
sounds good to me.

Reply:
@healther , yeah commit is another way to do it.

@gchavez2 hasn't clarified the reason for preferring git tarball URLs - so for now - I'll just merge this in - and add a new PR with sha256 sums - and worry about alternate formats later.
cc:  @xiaoyeli

Reply:
https://github.com/spack/spack/pull/9499